torch/csrc/jit/tensorexpr/analysis.h

```diff
@@ -46,6 +47,31 @@ class NodeFinder : public IRVisitor {
   std::vector<Node*> nodes;
 };
 
+// A class that analyzes the given program relevant for Block backend
+// It creates a map of multi dim buffers and their flat verions
+class CreateBufferMap : public IRVisitor {
+ public:
+  std::unordered_map<std::string, const Buf*> getBufferMap() const {
```

Comment:
Same here. Why return by value, instead of const reference?

Reply:
Thanks! fixed